life_expectancy/loading_saving.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+
+import pandas as pd
+
+
+def load_data(file_path: str) -> pd.DataFrame:
+    """
+    Method used to load data.
+
```

Comment:
Missing `file_path` in docstring.

Reply:
Resolved in this commit edf699ceba8e788b4631b448506ff52dea924679